azext_iot/central/_help.py

```diff
@@ -319,6 +320,92 @@ def _load_central_api_token_help():
     """
 
 
+def _load_central_cde_help():
+    helps[
+        "iot central app cde"
+    ] = """
+        type: group
+        short-summary: Create and Manage Continuous Data Export (CDE) for an application.
+        long-summary: Continuous Data Export enables exporting IoT Central measurements, devices, and device templates data to storage account.
+    """
+
+    helps[
+        "iot central app cde create"
+    ] = """
+        type: command
+        short-summary: Create a new continuous data export
+        examples:
+        - name: Add new continuous data export
+          text: >
+            az iot central app cde create
+            --app-id {appid}
+            --display-name {displayName}
+            --sources "devices, deviceTemplates, telemetry"
+            --enable True,
```

Comment:
With the default three state flag, you don't need to include the boolean value for true. Just `--enable` will return `True` for your command handlers arg.

Also you are able to use lowercase true/false.